Tests/SwiftStdlibTests/CharacterExtensionsTests.swift

```diff
@@ -68,6 +68,13 @@ final class CharacterExtensionsTests: XCTestCase {
     func testLowercased() {
         XCTAssertEqual(Character("S").lowercased, Character("s"))
     }
+    
+    func testRandom() {
+        let char1 = Character.random()
+        let char2 = Character.random()
+        
+        XCTAssertNotEqual(char1, char2)
```

Comment:
I understand the point of the test working this way, but it also causes to to randomly fail 🙂.

Reply:
A better way to do this is to accumulate 2 separate strings using `Character.random()` and then compare the two. The longer the string the less likely a collision.